script/cibuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```

Comment:
`--config _config.yml` is just the default afaict, so would prefer to leave as is.